src/main/java/com/greghaskins/spectrum/Spectrum.java

```diff
@@ -223,6 +224,30 @@ public static Configuration configure() {
 
 
   /**
+   * Implements a rules context within the current suite using the given rules class as a mix-in.
+   * These rules will cascade down and be applied at the level of specs or atomic specs.
+   * @param rulesClass type of object to create and apply rules to for each spec.
+   * @param <T> type of the object
+   * @return a supplier of the rules object
+   */
+  public static <T> Supplier<T> applyRules(final Class<T> rulesClass) {
```

Comment:
As discussed, we'll rename this one for better clarity. I'm still not sure where the best place for this entry point is, though. Not the worst if it's in root `Spectrum`, but as this is something I'll be marking as both experimental and JUnit-specific, it might make sense just letting people use `Rules.applyRules` directly, or maybe something like `JUnit.applyRules`. I'm flexible though.